app/api/resources/user.py

```diff
@@ -397,11 +388,13 @@ class LoginUser(Resource):
     @users_ns.response(HTTPStatus.OK, "Successful login", login_response_body_model)
     @users_ns.response(
         HTTPStatus.BAD_REQUEST,
-        "%s\n%s"
-        % (messages.USERNAME_FIELD_IS_MISSING, messages.PASSWORD_FIELD_IS_MISSING),
+        (
+            f"{messages.USERNAME_FIELD_IS_MISSING}\n"
+            f"{messages.PASSWORD_FIELD_IS_MISSING}"
+        ),
```

Comment:
Shouldn't we use a single line here, as done above

Reply:
Since you suggested the change and approved the PR too , do you want this change  and label will be updated as `Status changes requested` or I can update the label as `Ready to merge` ? @devkapilbansal

Reply:
I think we should make this change. If this needs to be done, then label should be changes requested, else it is ready for merging. What are your thoughts on this??